privacy_partner_report/views/privacy_report.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```

Comment:
:fire:

Reply:
It's v9

Reply:
v9 already supports `<odoo>` tag

Reply:
I know. But I will not make a new push just for this aesthetic change.

Reply:
Yeah, the point is that you don't need anymore to put a `<data>` inside a `<openerp>`.

BTW, [from v9 you can use `<data>`, `<openerp>` and  `<odoo>` interchangeably, it doesn't really matter](https://github.com/odoo/odoo/blob/80b325bbb2cb696caed41a180554553fe68bd6c7/openerp/import_xml.rng#L318-L349). IMHO, it's better to just use `<data>` and cover yourself against possible future rebrandings (we've had a few already).

Reply:
Doing that now, you avoid the need to change it when forward-porting. There are more than this change in the comments, so you can change all in one shot.

Reply:
I someday this PR is merged, I will do the automatic migration so, just don't bother with that.